PublicationsRouter/api/v3/Send.md

```diff
@@ -154,7 +159,7 @@ To do this, send the bare-minimum JSON notification, with only the format identi
 
         --FulltextBoundary
 
-        Content-Disposition: form-data; name="content"
+        Content-Disposition: name="content"
```

Comment:
I note that elsewhere in the file it still has the original: `Content-Disposition: form-data; name=...` which should presumably also be changed.